### DISCUSSION TITLE
打开程序报错

### DISCUSSION THREAD

**Discussion by swwyyy** [mikumifa/biliTickerBuy#110] [提问专区]
正常登陆之后打开程序就报错了想问问z怎么解决
![T12%}JEJLMRVIX@ 2U)8YD9](https://github.com/mikumifa/biliTickerBuy/assets/139413173/a5dd15b4-9343-448f-99ca-e49f969c6320)

**Comment by mikumifa** [comment]
没见过，哪个版本，多重启试试
